src/rect.rs

```diff
@@ -45,4 +52,12 @@ impl VelloVector for VelloRect {
             self.radius,
         )
     }
+
+    fn border_translation(&self, _time: f64) -> DVec2 {
+        DVec2::ZERO
+    }
+
+    fn border_rotation(&self, _time: f64) -> f64 {
+        0.0
+    }
```

Comment:
for `border_translation` we can do a naive approach of dividing the `time` into 4 sides of the rect:

```rs
// Prevent time from exceeding 1 so it loops over
let time = time % 1.0;
let rect_time = time * 4.0;

if rect_time > 3.0 {
    // return interpolated left_edge
} else if rect_time > 2.0 {
    // return interpolated bottom_edge
} else if rect_time > 1.0 {
    // return interpolated right_edge
} else {
    // return interpolated top_edge
}
```